### PR TITLE
index.bs,cosmetics: fix some typos

### PR DESCRIPTION
**Pull request by jzern**
just a quick pass, the text wasn't checked for coherency

***
<a href="https://pr-preview.s3.amazonaws.com/jzern/iamf/pull/231.html" title="Last updated on Jan 13, 2023, 1:25 AM UTC (d55effb)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/AOMediaCodec/iamf/231/d7bdaa1...jzern:d55effb.html" title="Last updated on Jan 13, 2023, 1:25 AM UTC (d55effb)">Diff</a>

### CODE REVIEW

**Comment by jzern** [comment]
@felicialim @sunghee-hwang please have a look.

'Decapsulate' and its other forms sounds strange to me. That may just be that I was expecting the more common mux/demux pairing.

File: index.bs [hunk 1]
```diff
@@ -287,7 +287,7 @@ class Bar {
 
  <b>UTF-8_Enc(label)</b>
  
- <dfn values noexport>UTF-8_Enc(label)</dfn> is byted represenation of the encoded <b>label</b>, which is UTF-8 string as defined in [[!RFC3629]], null terminated.
+ <dfn values noexport>UTF-8_Enc(label)</dfn> byte representation of the encoded <b>label</b>, which is UTF-8 string as defined in [[!RFC3629]], null terminated.
```
Comment (felicialim) [inline index.bs:290]: is the byte representation

**Review by felicialim — APPROVED** [review]

**Review by sunghee-hwang — APPROVED** [review]
LGTM

File: index.bs (hunk 1 above)
Comment (jzern) [reply index.bs:290]: I took the short form assuming this was a definition list and didn't need a full sentence, but I didn't look that closely at the generated output.

**Pull request merged**